doc/conf.py

```diff
@@ -0,0 +1,97 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../"))
+
+# -- Project information -----------------------------------------------------
+
+project = "Exasol Notebook Connector"
+copyright = "2025, Exasol"  # pylint: disable=redefined-builtin
+author = "Exasol"
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    "sphinx.ext.todo",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.viewcode",
+    "sphinx.ext.napoleon",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.autosummary",
+    "sphinx_copybutton",
+    "myst_parser",
+    "sphinx_design",
+    "sphinx_inline_tabs",
+    "sphinx_copybutton",
+    "exasol.toolbox.sphinx.multiversion",
+]
+
+# Restrict versions to include into multiversion documentation
+# smv_tag_whitelist = r'^[^012].*$'
```

Comment:
Added

Reply:
But generating multi-doc currently still causes errors as Sphinx configuration is missing on old tags but also on main branch.

Reply:
I get that once this is merged to main that the multiversion documentation should work for main, but what's the plan for the older tags? 

Would we not publish them (like modify the `.github/workflows/build-and-publish.yml`, reject them with a to-be-developed PTB option, etc.)? It might be possible when building the docs to inject the conf.py.

Reply:
I have the same questions.